### PR TITLE
Dynamo table update

### PR DESCRIPTION
**Pull request by diana-macrometa**
Added a topic explaining how to create a Dynamo Table collection. Part of DOC-299 import.

### CODE REVIEW

File: docs/collections/dynamo/create-dynamo-table.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Create a Dynamo Table
+sidebar_position: 10
+---
+
+Dynamo Table collection is available for AWS Dynamo Database users. Macrometa’s platform can be used as a data store to acquire Global data distribution within milliseconds. To switch to GDN, the user need to change the connection URL, AccessKey, and SecretKey. AWS Dynamo DB SDK and CLI are compatible. Geo Distribution is set to Global by default.
```
Comment (jamesm-macrometa) [inline docs/collections/dynamo/create-dynamo-table.md]:
```suggestion
AWS Dynamo Database users can create Dynamo Table collections. You can use Macrometa as a datastore and achieve global data distribution within milliseconds. 

To switch to GDN, you must change the connection URL, `AccessKey`, and `SecretKey`. AWS Dynamo DB SDK and CLI are compatible. Geo-distribution is global by default.
```

File: docs/collections/dynamo/create-dynamo-table.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Create a Dynamo Table Collection  with the Console
+
+Follow these instructions to create a new collection using the GDN console web UI.
```
Comment (jamesm-macrometa) [inline docs/collections/dynamo/create-dynamo-table.md]:
```suggestion
To create a Dynamo Table collection:
```

File: docs/collections/dynamo/create-dynamo-table.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: Create a Dynamo Table
+sidebar_position: 10
+---
+
+Dynamo Table collection is available for AWS Dynamo Database users. Macrometa’s platform can be used as a data store to acquire Global data distribution within milliseconds. To switch to GDN, the user need to change the connection URL, AccessKey, and SecretKey. AWS Dynamo DB SDK and CLI are compatible. Geo Distribution is set to Global by default.
+
+## Create a Dynamo Table Collection  with the Console
```
Comment (jamesm-macrometa) [inline docs/collections/dynamo/create-dynamo-table.md]:
```suggestion
## Create a Dynamo Table Collection with the Console
```

File: docs/collections/dynamo/create-dynamo-table.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Follow these instructions to create a new collection using the GDN console web UI.
+
+1. Log in to your Macrometa account.
```
Comment (jamesm-macrometa) [inline docs/collections/dynamo/create-dynamo-table.md:14]:
```suggestion
1. Log into your Macrometa account.
```

**Review by jamesm-macrometa — CHANGES_REQUESTED** [review]
A few revisions

File: docs/collections/dynamo/create-dynamo-table.md (hunk 4 above)
Comment (diana-macrometa) [reply docs/collections/dynamo/create-dynamo-table.md:14]: The original is actually correct. Weird, I know.

Comment (jamesm-macrometa) [reply docs/collections/dynamo/create-dynamo-table.md:14]: > The original is actually correct. Weird, I know.

I always thought it was more of a preference thing, but either way works for me.  Will remember this!

**Review by jamesm-macrometa — APPROVED** [review]

**Pull request merged**